### DISCUSSION TITLE
nillion-nada-app

### DISCUSSION THREAD

**Discussion by andreydos** [NillionNetwork/discussions#98] [Show and tell]
### Show and tell project type

Other

### Github Repo Link

https://github.com/andreydos/nillion-nada-app

### Project Description

This is my first application built on the Nillion blockchain. Prior to this project, I had no experience with blockchain technology. I am excited to share that simple example. And it works!

### What problems does your project solve? How does it preserve privacy for users?

Problem 1: No understanding of what Nillion is. Solution: Now I understand something!

**Comment by Davetbutler** [comment]
Hey @andreydos.

Dave from the product team here. Great to see you got started on Nillion. Have you got ideas on what to build next? [Maybe you would be interested in our builder bounties](https://github.com/orgs/NillionNetwork/projects/3/views/1)

I would love to learn two things, to help us improve Nillon for everyone:
1. Where did you hear about Nillion?
2. How did you find the onboarding experience, what were the major pain points (if any)?
